test/src/main/java/org/openjdk/skara/test/TestBotFactory.java

```diff
@@ -156,4 +156,77 @@ public JSONObject specific() {
         }
         throw new RuntimeException("Failed to find bot factory with name: " + name);
     }
+
+    public List<Bot> createBots(String name, JSONObject configuration) {
```

Comment:
This is a copy of the method above. They should share implementation. You can make `create` call `createBots` and keep lines 151 to 154 to convert the list.

Reply:
> This is a copy of the method above. They should share implementation. You can make `create` call `createBots` and keep lines 151 to 154 to convert the list.

Ok, but the implementations of `continuousIntegration` and `repository` in `BotConfiguration` are a little different between `create` and `createBots`. Hope it will not affect other tests, althoughI don't think it's going to have any bad effects.

Reply:
Ah I didn't realize you changed them. Ideally we should be able to have a unified implementation.